src/registrar/models/domain_request.py

```diff
@@ -644,6 +644,33 @@ class RejectionReasons(models.TextChoices):
         null=True,
         blank=True,
         help_text="Anything else?",
+        verbose_name="Additional details",
+    )
+
+    # This is a drop-in replacement for a has_anything_else_text() function.
+    # In order to track if the user has clicked the yes/no field (while keeping a none default), we need
+    # a tertiary state. We should not display this in /admin.
+    has_anything_else_text = models.BooleanField(
+        null=True,
+        blank=True,
+        help_text="Determines if the user has a anything_else or not",
+    )
+
+    cisa_representative_email = models.EmailField(
+        null=True,
+        blank=True,
+        db_index=True,
+        verbose_name="CISA region representative",
+        max_length=320,
+    )
+
+    # This is a drop-in replacement for an has_cisa_representative() function.
+    # In order to track if the user has clicked the yes/no field (while keeping a none default), we need
+    # a tertiary state. We should not display this in /admin.
+    has_cisa_representative = models.BooleanField(
```

Comment:
Same, I think this should be a method similar to is_restricted or whatever we called it for users

Reply:
Got it